src/helpers/graphqlHelpers.ts

```diff
@@ -153,23 +163,31 @@ export const getPrimaryOperationForGetRequest = (
     return null
   }
 
+  const queryParam = details.request.queryString.find(
+    (qs) => qs.name === "query"
+  )
+
+  const query = queryParam && decodeQueryParam(queryParam.value)
```

Comment:
Since the `&&` logic is repeated here. Maybe better to move logic to `decodeQueryParam`. We would also not need to double not `!!query` in that case. Since `decodeQueryParam` can return the value or null.

Reply:
IMO, I prefer the `&&` pattern because it enforces types on the `decodeQueryParam` to have a value. Also, why bother executing a try/catch if you are sure that the value is null/undefined?